ERCS/erc-7683.md

```diff
@@ -51,42 +51,55 @@ struct CrossChainOrder {
 	/// or any other order-type specific information
 	bytes orderData;
 }
+
+/// @title OnchainCrossChainOrder CrossChainOrder type
+/// @notice Standard order struct for user-initiated orders, where the user is the msg.sender.
+struct OnchainCrossChainOrder {
```

Comment:
> at least make the param ordering the same so it's clear what the diff is

Good call. Meant to do this. Probably just a mistake.

> thoughts on composing gasless + onchain given the former is a superset of the latter?

I originally wanted to do it this way. Problem was that it makes it more confusing to read and annoying to construct since there's an extra layer of nesting. Not obvious to me that nesting makes things easier for the people building and touching this structure offchain.

Reply:
Corrected the ordering.